common/test/acceptance/tests/lms/test_ccx.py

```diff
@@ -42,5 +42,37 @@ def test_create_ccx(self):
         self.coach_dashboard_page.fill_ccx_name_text_box(ccx_name)
         self.coach_dashboard_page.wait_for_page()
 
+
+@attr('shard_7')
+class CreateCCXCoachTest(BaseCCXCoachTest):
+    """
+    Test ccx end to end process.
+    """
+    def test_create_ccx(self):
+        """
+        Assert that ccx created.
+        """
+        self.create_ccx()
         # Assert that new ccx is created and we are on ccx dashboard/enrollment tab.
         self.assertTrue(self.coach_dashboard_page.is_browser_on_enrollment_page())
+
+
+@attr('a11y')
+class ScheduleTabA11yTest(BaseCCXCoachTest):
+    """
+    Class to test schedule tab accessibility.
+    """
+
+    def test_schedule_tab_a11y(self):
+        """
+        Test schedule tab accessibility.
+        """
+        self.create_ccx()
+        schedule_section = self.coach_dashboard_page.select_schedule()
+
+        schedule_section.a11y_audit.config.set_rules({
+            'ignore': [
+                'link-href', 'skip-link'  # TODO: AC-233
```

Comment:
Are these ignores really necessary because of OAuth settings (which is what https://openedx.atlassian.net/browse/AC-233 refers to)? I only include ignores if they are necessary, and if they are related to something different from AC-233, create a new accessibility ticket.